packages/common-library/src/common_library/user_messages.py

```diff
@@ -0,0 +1,11 @@
+def user_message(msg: str, *, _version=None) -> str:
+    """Marks a message as user-facing
+
+    Arguments:
+        msg -- human-friendly string that follows docs/user-messages-guidelines.md
+        _version -- version number to track changes to messages; increment when modifying an existing message
+
+    Returns:
+        The original message string, allowing it to be used inline in code
+    """
+    return msg
```

Comment:
Please do it this way, it looks les awkward in code.

```suggestion
def user_message(msg: str, *, version=None) -> str:
    """Marks a message as user-facing

    Arguments:
        msg -- human-friendly string that follows docs/user-messages-guidelines.md
        _version -- version number to track changes to messages; increment when modifying an existing message

    Returns:
        The original message string, allowing it to be used inline in code
    """
    _ = version
    return msg
```